src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -1134,8 +1136,25 @@ public void visitSources(SCMSourceObserver observer) throws IOException, Interru
                     } else {
                         repositories = org.listRepositories(100);
                     }
+
+                    List<String> repoList = StreamSupport.stream(repositories.spliterator(), false)
+                            .map(GHRepository::getName)
+                            .collect(Collectors.toList());
+
+                    String repoString = String.join(",", repoList);
```

Comment:
NIT, slightly inefficient vs creating the string without the intermediate map (e.g. untested)


```suggestion
                    String repoString  = StreamSupport.stream(repositories.spliterator(), false)
                            .map(GHRepository::getName)
                            .collect(Collectors.joining(", "));
```